projects/step-frontend/src/lib/modules/execution/components/execution-progress/execution-progress.component.scss

```diff
@@ -1,5 +1,5 @@
 as-split {
-  height: 80vh;
+  height: calc(100vh - 134px);
```

Comment:
use flexbox, it is possible to achieve the same with flexbox, as-split should have flex 1

Reply:
@bojidaryovchev It's required to specify the concrete height, so angular-split could draw dragging bar properly

Reply:
@bojidaryovchev it was my first instinct too. I think theoretically it is possible, but the entire page should be restyled in order to at least try it.